src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -127,10 +132,12 @@ function processElement(cellElement) {
 			$(npcNameElement).addClass('processedNPC knownNPC');
 			$(npcFastAttackElement).css('color', '#006400');
 			$(npcFastAttackElement).append(' ( -' + lifeLoss + ' LP, = ' + (playerExpectedLife - lifeLoss) + ' LP )');
-			const randomWait = Math.floor(Math.random() * (2000 - 500 + 1) + 500);
-			setTimeout(function() {
-				// $(npcFastAttackElement).trigger('click'); - use this at your own risk...
-			}, randomWait);
+			// Npc auto attack if enable
+			if (npcAutoAttack == true) {
```

Comment:
u can leave out the `== true`, i.e. just `if (foo)`, not `if (foo == true)`. other than that, cool stuff :)

Reply:
Danke, wieder was gelernt :)

beide if's gefixxt